h2/src/main/org/h2/jdbc/JdbcLob.java

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright 2004-2018 H2 Group. Multiple-Licensed under the MPL 2.0,
+ * and the EPL 1.0 (http://h2database.com/html/license.html).
+ * Initial Developer: H2 Group
+ */
+package org.h2.jdbc;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.PipedInputStream;
+import java.io.PipedOutputStream;
+import java.io.Reader;
+import java.io.Writer;
+import java.sql.SQLException;
+
+import org.h2.api.ErrorCode;
+import org.h2.message.DbException;
+import org.h2.message.TraceObject;
+import org.h2.util.IOUtils;
+import org.h2.util.Task;
+import org.h2.value.Value;
+
+/**
+ * Represents a large object value.
+ */
+public abstract class JdbcLob extends TraceObject {
+
+    final class Output extends PipedOutputStream {
```

Comment:
can we give this a more useful name? "Output" is rather generic.

LobPipedOutputStream?